apps/nextjs/src/app/[locale]/manage/users/[userId]/_components/profile.accordion.tsx

```diff
@@ -5,22 +5,46 @@ import { Button, Stack, TextInput } from "@mantine/core";
 import type { RouterOutputs } from "@homarr/api";
 import { clientApi } from "@homarr/api/client";
 import { useForm, zodResolver } from "@homarr/form";
+import {
+  showErrorNotification,
+  showSuccessNotification,
+} from "@homarr/notifications";
 import { useI18n } from "@homarr/translation/client";
 import { validation } from "@homarr/validation";
 
 import { revalidatePathAction } from "~/app/revalidatePathAction";
+import { ErrorDisplay } from "~/components/utils";
 
 interface ProfileAccordionProps {
   user: NonNullable<RouterOutputs["user"]["getById"]>;
 }
 
 export const ProfileAccordion = ({ user }: ProfileAccordionProps) => {
   const t = useI18n();
-  const { mutate, isPending } = clientApi.user.editProfile.useMutation({
-    onSettled: async () => {
-      await revalidatePathAction("/manage/users");
-    },
-  });
+  const { mutate, isPending, isError, error } =
+    clientApi.user.editProfile.useMutation({
+      onError(error) {
+        showErrorNotification({
+          title: t(
+            "management.page.user.edit.section.profile.editProfile.title",
+          ),
+          message: error.message,
```

Comment:
I would like it more if we would show the user a specific error message that we can translate

Reply:
This handles every error, I think translating server-side error messages a bit too overkill (the already user exist comes from the procedure, I also got a "fetch failed" error when the server was killed)

Reply:
Yes but errors that we handle ourself (Like existing user) would make sense to give the user feedback in their own language